### PR TITLE
feat: influx multi page query

### PR DESCRIPTION
**Pull request by eivindkopperud**
Allows querying for several ids. One query fetches all views for the given ids, while the other one fetches the average page view for those pages in the given time interval.

Flux allows for contains() filtering (https://community.grafana.com/t/grafana-influxdb-flux-query-for-displaying-multi-select-variable-inputs/35536/13), but this is not efficient when
fields are queried for. Therefore, I introduce some regex logic which was proposed in the Grafana discussion. The regex is on the form ^(x|y|z).

Also, the .get_fields() was removed since the aggregated version does not have fields, as I group on measurements.

### CODE REVIEW

File: insert_influx.py [hunk 1]
```diff
@@ -78,14 +78,44 @@ def do_job(write_api, filepath):
...
+    (f'from(bucket: "{bucket}")'
+       '|> range(start: timestart, stop: timestop)'
+       '|> filter(fn: (r) => r["_measurement"] == "pageview")'
+       '|> filter(fn: (r) => r["_field"] == "hits")'
+       f'|> filter(fn: (r) => r["page_id"] =~ {regex})'
+       '|> sum(column: "_value")'
+       '|> group(columns: ["_measurement"], mode:"by")'
+       '|> mean(column: "_value")')
+
+def parse_multiple_id_regex(ids):
```
Comment (TheodorRene) [inline insert_influx.py:105]: Tror jeg aldri kommer til å forstå performancen til regex. Men ser i forumposter du lenket til at dette er nok den mest effektive måten å gjøre det på. veldig nice!

File: insert_influx.py [hunk 2]
```diff
@@ -78,14 +78,44 @@ def do_job(write_api, filepath):
...
+    return f"/^({pipe_separated_ids})$/"
+
 def do_query(query_api, query, params):
     debuglog("doing query " +  query)
     result = query_api.query(org=org,  query=query, params=params)
 
     results = []
     for table in result:
-            for record in table.records:
-                        results.append((record.get_field(), record.get_value()))
```
Comment (TheodorRene) [inline insert_influx.py:88]: enig, som oftest trengre man ikke get_field. Det setter man gjerne i applikasjonslaget

**Review by TheodorRene — APPROVED** [review]

**Comment by TheodorRene** [comment]
LGTM

File: insert_influx.py (hunk 1 above)
Comment (eivindkopperud) [reply insert_influx.py:105]: Haha, ja, veldig snodig. Prøvde å matche på én id med contains(), men da timeoutet requesten :joy_cat:

**Pull request merged**